### PR TITLE
C++

### PR DESCRIPTION
**Pull request by alexeybelkov**
fixed microbug in interp2d

### CODE REVIEW

File: imops/interp2d.py [hunk 1]
```diff
@@ -81,7 +81,7 @@ def __call__(self, points: np.ndarray, values: np.ndarray = None, fill_value: fl
         points: np.ndarray
             2-D array of data point coordinates to interpolate at
         values: np.ndarray
-            1-D array of fp32/fp64 values to use at initial points. If passed, existing values will be rewritten
+            1-D array of fp32/fp64 values to use at initial points. If passed, existing values will NOT be rewritten
```
Comment (maxme1) [inline imops/interp2d.py:84]: "If passed, existing values will NOT be rewritten" - you don't need to specify this. This is what the user expects anyway

File: imops/interp2d.py [hunk 2]
```diff
@@ -90,16 +90,16 @@ def __call__(self, points: np.ndarray, values: np.ndarray = None, fill_value: fl
         new_values: np.ndarray
             interpolated values at given points
         """
-        self.values = values or self.values
+        x_values = values if values is not None else self.values
```
Comment (maxme1) [inline imops/interp2d.py:93]: why not just 
```python
if values is None:
  values = self.values
```
? the new variable seems redundant

Comment (alexeybelkov) [reply imops/interp2d.py]: fixed

File: imops/interp2d.py (hunk 1 above)
Comment (alexeybelkov) [reply imops/interp2d.py]: fixed

**Review by maxme1 — APPROVED** [review]

**Pull request merged**